m4/awscli_help.m4

```diff
@@ -33,8 +33,9 @@ Optional arguments:
                           Specify type of AWS CLI installation. Options are:
                           "portable-exe", "system-sandbox" (default is
                           "system-sandbox")
-  --with-download-deps    Download all dependencies and use those when
-                          building the AWS CLI. If not specified, the
+  --with-download-deps    Download all Python dependencies and use those when
```

Comment:
It would be great if we can indicate that currently it only downloads Python dependencies but this option might be expanded in the future to other dependencies. Maybe we can add the disclaimer at the end of the help that the dependencies downloaded could be expanded beyond Python dependencies?

Reply:
I think it makes more sense to just document what it does now. If we change it we change the string.